cmd/snap/cmd_add_cap.go

```diff
@@ -73,10 +72,12 @@ var (
 )
 
 func init() {
-	_, err := parser.AddCommand("add-cap", shortAddCapHelp, longAddCapHelp, &cmdAddCap{})
-	if err != nil {
-		logger.Panicf("unable to add add-caps command: %v", err)
-	}
+	commands = append(commands, cmdInfo{
```

Comment:
still have a mild preference for this being at least spelled addCommand(cmdInfo{...})